nservicebus/hosting/azure-functions/service-bus_configuration_ASBFunctions_[,1.2].partial.md

```diff
@@ -0,0 +1,3 @@
+NServiceBus can be registered and configured on the host builder using the `UseNServiceBus` extension method in the startup class:
```

Comment:
As far as I can tell, this is the same in both partials. That's not really an issue but it could be pulled up into the main file,

Reply:
moved to the parent file.